pkg/kms/localkms/localkms.go

```diff
@@ -158,6 +163,44 @@ func getKeyTemplate(keyType kms.KeyType) (*tinkpb.KeyTemplate, error) {
 	}
 }
 
+func ecdsaP256KeyWithoutPrefixTemplate() *tinkpb.KeyTemplate {
+	return createECDSAKeyTemplate(commonpb.HashType_SHA256,
+		commonpb.EllipticCurveType_NIST_P256,
+		ecdsapb.EcdsaSignatureEncoding_IEEE_P1363,
```

Comment:
for those interested,

new key templates with `ecdsapb.EcdsaSignatureEncoding_IEEE_P1363` encoding format is needed to support exporting marshalled elliptic curve keys since the default Tink templates use `ecdsapb.EcdsaSignatureEncoding_DER` format. This encoding change is needed to allow DID verification using public signing keys imported to Tink keyset.Handle instances.

Reply:
Thank you for the explanation, @Baha-sk !